composites/Plugin/Synonyms/reducers/synonyms.js

```diff
@@ -0,0 +1,77 @@
+import {
+	SET_SYNONYMS,
+	ADD_SYNONYMS,
+	INSERT_SYNONYMS,
+	CHANGE_SYNONYMS,
+	REMOVE_SYNONYMS,
+} from "../actions/synonyms";
+
+const INITIAL_STATE = [];
+
+/**
+ * Checks if an index is inside the array.
+ *
+ * @param {array}  arr   The array to check in.
+ * @param {number} index The index to check for.
+ *
+ * @returns {boolean} Whether the index exists in the array or not.
+ */
+function indexInArray( arr, index ) {
```

Comment:
Looks like this is indexNotInArray.

Reply:
I would avoid negations in function names.

Reply:
Oh definitely. I would align the boolean with the function title though. `indexInArray` outputs `true` if it's not in array, which is odd... So I'd return the opposite value.

Reply:
I also know you know 😝